make_pyne_docker_image.sh

```diff
@@ -0,0 +1,112 @@
+#! /bin/bash
+
+#_____________________________________________________________________________
+# usage
+# call if option -h, -help or --help, display usage and quit
+function usage ()
+{
+echo "---------------------------------------------------------------------------"
+echo "---------------------- make_pyne_docker_image usage: ----------------------"
+echo " "
+echo "  -h|-help|--help: print help/usage "
+echo " "
+echo "  --moab|-moab|moab : add MOAB to the build "
+echo " "
+echo "  --pymoab|-pymoab|pymoab : add pyMOAB to the build (also enable MOAB) "
+echo " "
+echo "  --dagmc|-dagmc|dagmc : add DAGMC to the build (also enable MOAB) "
+echo " "
+echo "  --deps-only|-deps-only|deps-only|--deps|-deps|deps : only build "\
+    "dependencies (PyNE will not be build/installed)"
+echo " "
+echo "  -tag=*|--tag=*|tag=*|-t=*|--t=*|t=*: set the tag name of the "\
+            "created docker image, if not provided tag name will be form from "\
+            "the install option : 
+            \"ubuntu_18.04(_pyne/_pyne-deps)(moab/pymoab/)(dagmc/) \""
+echo " "
+echo "---------------------------------------------------------------------------"
+exit 418 
+}
+
+
+# Parsing arguments
+MOAB=false
+DAGMC=false
+PYMOAB=false
+PYNE=true
+
+for arg in "$@"; do
+    case $arg in
+        -h|-help|--help )
+            usage 
+            ;;
+        --moab|-moab|moab )
+            MOAB=true
+            ;;
+        --dagmc|-dagmc|dagmc )
+            DAGMC=true 
+            MOAB=true 
+            ;;
+        --pymoab|-pymoab|pymoab )
+            PYMOAB=true 
+            MOAB=true 
+            ;;
+        --deps-only|-deps-only|deps-only )
+            PYNE=false 
+            ;;
+        --deps|-deps|deps )
+            PYNE=false 
+            ;;
+        -tag=*|--tag=*|tag=*|-t=*|--t=*|t=*)
+            img_tag="${arg#*=}"
+            shift # past argument=value
+            ;;
+    esac
+done
+
+# Parse option, form arguments
+pyne_option='_pyne'
+if  $MOAB; then
+    moab_arg='--build-arg build_moab=YES'
+    moab_option='_moab'
+fi
+if  $DAGMC; then
+    dagmc_arg='--build-arg build_dagmc=YES'
+    dagmc_option='_dagmc'
+    moab_option=''
+fi
+if  $PYMOAB; then
+    pymoab_arg='--build-arg enable_pymoab=YES'
+    pymoab_option='_pymoab'
+    moab_option-''
+fi
```

Comment:
For the PYMOAB and DAGMC args, should probably check that MOAB is also enabled. Can't build DAGMC/PYMOAB without MOAB.

Reply:
it is checked when parsing the arguments and in the dockerfile